AcademyResidentInformationApi/V1/Gateways/IAcademyGateway.cs

```diff
@@ -9,7 +9,7 @@ public interface IAcademyGateway
     {
         List<ClaimantInformation> GetAllClaimants(int cursor, int limit, string firstname = null,
             string lastname = null, string postcode = null, string address = null);
-        ClaimantInformation GetClaimantById(int claimId, int personRef);
```

Comment:
I think we want `claimId` and `personRef` and not `claimid`, `houseId` and `memberId`.

`claimid`, `houseId` and `memberId` would give us a unique record but not a necessarily a person we could track.

I think we want to track the person so using `claimId` and `personRef` should help us with instead.